DARPA-ASKEM/terarium
# Design change document node

# Description

Design Change

- Add notification progress bar to the document Node
- Add a thumbnail of the Pdf document

![image](https://github.com/user-attachments/assets/09689621-213b-4c0f-b0e6-946d4e7e94a1)

Testing:

- Start up local build
- Create new workflow and upload a new doc.
- The thumbnail should appear
- The progress bar should also appear after a moment when the extraction starts
